python/llm/example/GPU/HuggingFace/Multimodal/qwen2-audio/README.md

```diff
@@ -0,0 +1,130 @@
+# Qwen2-Audio
+In this directory, you will find examples on how you could apply IPEX-LLM INT4 optimizations on Qwen2-Audio models on [Intel GPUs](../../../README.md). For illustration purposes, we utilize the following model: 
+- [Qwen/Qwen2-Audio-7B-Instruct](https://huggingface.co/Qwen/Qwen2-Audio-7B-Instruct)
+
+as reference model.
+
```

Comment:
we utilize [Qwen/Qwen2-Audio-7B-Instruct](https://huggingface.co/Qwen/Qwen2-Audio-7B-Instruct) as reference model.